### PR TITLE
Droth 649 fix get by municipality methods

### PR DESCRIPTION
**Pull request by leirasf**

### CODE REVIEW

File: digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/ObstacleService.scala [hunk 1]
```diff
@@ -24,14 +24,14 @@ class ObstacleService(val roadLinkService: RoadLinkService) extends PointAssetOp
   override def create(asset: IncomingObstacle, username: String, geometry: Seq[Point], municipality: Int, administrativeClass: Option[AdministrativeClass] = None): Long = {
     val mValue = GeometryUtils.calculateLinearReferenceFromPoint(Point(asset.lon, asset.lat, 0), geometry)
     withDynTransaction {
-      OracleObstacleDao.create(asset, mValue, username, municipality)
+      OracleObstacleDao.create(asset, mValue, username, municipality, VVHClient.createVVHTimeStamp(5))
```
Comment (halied) [inline digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/ObstacleService.scala]: What is this parameter 5?

**Comment by halied** [comment]
Ok

**Comment by joanacoelho** [comment]
Looks ok

**Comment by marquesfm** [comment]
OK for me

**Pull request merged**